Documentation/FrameworkOverview.md

```diff
@@ -101,6 +127,7 @@ On OS X, RAC adds a `rac_command` property to
 [NSButton][NSButton+RACCommandSupport] for setting up these behaviors
 automatically.
 
+<!-- TODO: Remove? Or is there a replacement / is startWithSignal important enough to talk about it here? -->
```

Comment:
This should be deleted. Not sure if it's worth mentioning `startWithSignal` at this high a level